docs/source/content/supervised/bayes.rst

```diff
@@ -0,0 +1,180 @@
+existing classes and what features they have. In the example above, we choose
+the class that most resembles our input as its classification. This
+technique is based around using Bayes’ Theorem. If you’re unfamiliar with what
+Bayes’ Theorem is, don’t worry! We will explain it in the next section.
+
+
+**************
+Bayes’ Theorem
+**************
+Bayes’ Theorem [*Equation 1*] is a very useful result that shows up in
+probability theory and other disciplines.
+
+.. figure:: _img/Bayes.png
+
+   **Equation 1. Bayes' Theorem**
+
+With Bayes’ Theorem, we can examine conditional probabilities (the probability
+of an event happening given another event has happened). P(A|B) is the
+probability that event A will happen given that event B has already happened.
+We can determine this value using other information we know about events A and
+B. We need to know P(B|A) (the probability that event B will happen given that
+event A has already happened), P(B) (the probability event B will happen), and
+P(A) (the probability event A will happen). We can even apply Bayes’ Theorem
+to machine learning problems!
+
+
+***********
+Naive Bayes
+***********
+Naive Bayes classification uses Bayes’ Theorem with some additional
+assumptions. The main thing we will assume is that features are independent.
+Assuming independence means that the probability of a set of features
+occurring given a certain class is the same as the product of all the
+probabilities of each individual feature occurring given that class. In the
+case of our fruit example above, being red does not affect the probability of
+being moderately sized so assuming independence between color and size is
+fine. This is often not the case in real-world problems where features may
+have complex relationships. This is why “naive” is in the name. If the math
+seems complicated, don’t worry! The code will handle the number crunching for
+us. Just remember that we are assuming that features are independent of each
+other to simplify calculations.
+
+In this technique, we take some input and calculate the probability of it
+happening given that it belongs to one of our classes. We must do this for
+**each** of our classes. After we have all these probabilities, we just take
+the one that’s the largest as our prediction for what class the input belongs
+to.
+
+
+**********
+Algorithms
+**********
+Below are some common models used for Naive Bayes classification. We have
+separated them into two general cases based on what type of feature
+distributions they use: continuous or discrete. Continuous means real-valued
+(you can have decimal answers) and discrete means a count (you can only have
+whole number answers). Also provided are the relevant code snippets for each
+algorithm.
+
+Gaussian Model (Continuous)
+===========================
+Gaussian models assume features follow a normal distribution. As far as you
+need to know, a normal distribution is just a specific type of distribution.

```

Comment:
We should let the user know briefly what each of these categories are. For instance, a gaussian distribution is one where most of the values fall towards the mean (maybe have a picture too)